### ISSUE TITLE
Missing https and http packages (React-Native)

### ISSUE THREAD

**Issue by gitteraz** [algolia/search-insights.js#229]
Hi,
I have been trying to to use this package with react-native expo without success due to missing node https and http in react-native environment. I have tried a couple of shim libraries also without success.

Kindly point me in the right direction. I have implemented Instant-Search successfully all that is missing is tracking clicks.

Thanks.

**Comment by Haroenv** [comment]
If you alias `http` and `https` to empty modules (like this package does by default normally), it should be working as expected. Do you have specific webpack / metro / babel configuration that could have changed it? 

Could you reproduce this on a GitHub repository?

When I last tried it (code here: https://github.com/algolia/react-instantsearch/tree/chore/rn-example-insights/examples/react-native) our aliasing worked

**Comment by theohdv** [comment]
@Haroenv I've tried on the repo you gave: https://github.com/algolia/react-instantsearch/tree/chore/rn-example-insights/examples/react-native and it doesn't work.

I got the same problem on my repo (without expo and with the latest version of RN).

Here is a screenshot of the error:
![Screenshot 2020-10-06 at 15 17 35](https://user-images.githubusercontent.com/26060802/95206569-24bae500-07e7-11eb-887b-d986c0780ef7.png)

**Comment by theohdv** [comment]
this doesn't work too, it fails during yarn install:
<img width="880" alt="Screenshot 2020-10-07 at 11 23 02" src="https://user-images.githubusercontent.com/26060802/95312695-89ca1580-088f-11eb-8bb5-eb2d239c89d8.png">

**Comment by theohdv** [comment]
And even if I alias http and https to empty modules. The lib won't work because this is the only way to make request using node:
![Screenshot 2020-10-07 at 14 16 50](https://user-images.githubusercontent.com/26060802/95331844-dcfd9180-08aa-11eb-90c7-fd68191eb042.png)

**Comment by Haroenv** [comment]
sorry, the empty module should be a folder with:

package.json:

```json
{
  "name": "empty-module",
  "version": "0.0.0"
}
```

index.js
```js
module.exports = {};
```

and installed as:

```json
"dependencies": {
    "https": "file:./empty-module",
    "http": "file:./empty-module"
}
```

The node requester isn't needed for react native, it uses XHR, but metro seems to be very strict about resolving all dependencies, even if only their presence is checked.

**Comment by theohdv** [comment]
Thanks for the support,
I did exactly what you told me. When i'm trying to do:
```
    searchInsights.init(
      config().getString('algolia_api_key'),
      config().getString('algolia_app_id')
    );
```

I got this error:
![Simulator Screen Shot - iPhone 11 - 2020-10-08 at 09 31 44](https://user-images.githubusercontent.com/26060802/95428276-197fca80-0949-11eb-8dd0-82fc460ef5d6.png)

**Comment by Haroenv** [comment]
Sorry if that was unclear, but if you use version 1.6.2 of search insights (I've just pushed to the example repo) it works fully for me, no need for http / https workarounds in your client side.

**Comment by eunjae-lee** [comment]
fixed in #220
